orca-web/src/main/groovy/com/netflix/spinnaker/orca/keel/model/OrchestrationViewModel.groovy

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.netflix.spinnaker.orca.model
+package com.netflix.spinnaker.orca.keel.model
```

Comment:
I don't think we want this?

Reply:
Don't know how that snuck in there!! Good catch.